src/official-account/simple-unirest.ts

```diff
@@ -22,11 +24,12 @@ export interface SimpleUnirest {
 }
 
 function getSimpleUnirest (
-  endpoint : string,
+  endpoint : string, headers?: object
```

Comment:
Add custom headers to unirest.

Reply:
Could you please explain the reason why we need to add this?

Reply:
When we upload file,  there is a different header config with post json data.  Maybe there is another header config different with json and file header format.  This is my consideration.  How do you think about?

Reply:
Please create an issue and describe it in details.

Thank you very much.

Reply:
Uploading file has passed with SimpleUnirest tools, I will revert this.